.github/workflows/deploy-pr-preview.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy pr preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - closed
+    paths:
+      - "docs/sources/**"
+
+jobs:
+  deploy-pr-preview:
+    if: github.repository == 'grafana/mimir'
+    uses: grafana/writers-toolkit/.github/workflows/deploy-preview.yml@main
+    with:
+      sha: ${{ github.event.pull_request.head.sha }}
+      branch: ${{ github.head_ref }}
+      event_number: ${{ github.event.number }}
+      title: ${{ github.event.pull_request.title }}
+      repo: mimir
+      website_directory: content/docs/mimir/latest
+      relative_prefix: /docs/mimir/latest/
```

Comment:
this repo also hosts the mimir-distributed helm chart docs under https://github.com/grafana/mimir/tree/3050e72641187ecaf84792036d23843d37f8cbce/docs/sources/helm-charts/mimir-distributed

does this config prevent from rendering those docs too?

Reply:
Well remembered, I think we'll need to modify and release a v2 of the workflow that supports multiple source directories.

Reply:
In the mean time 
```suggestion
      source_directory: docs/sources/mimir
      website_directory: content/docs/mimir/latest
      relative_prefix: /docs/mimir/latest/
```